Modules/Scripted/EditorLib/HelperBox.py

```diff
@@ -165,19 +165,17 @@ def setVolumes(self,masterVolume,mergeVolume):
 
   def setMasterVolume(self,masterVolume):
     """select merge volume"""
+    if isinstance(masterVolume, basestring):
+      masterVolume = slicer.mrmlScene.GetNodeByID(masterVolume)
     self.masterSelector.setCurrentNode( masterVolume )
-    self.select()
```

Comment:
Agreed. That is the observed behaviour after this change is implemented. 

@fedorov: I would be happy to learn more about your use case to improve/fix the existing interface.